src/stateAggregator/accessors/aliasAccessor.ts

```diff
@@ -116,19 +115,21 @@ export class AliasAccessor extends AsyncOptionalCreatable {
    */
   public async setAndSave(alias: string, entity: Aliasable): Promise<void> {
     // get a very fresh copy to merge with to avoid conflicts, then lock
-    await this.readFileToAliasStore(true);
+    const lockResponse = await lockInit(this.fileLocation);
```

Comment:
this is the biggest change.  I reworked alias to use the same exported lockfile that everything else is using.